Comp2024/src/main/java/frc/robot/lib/math/Conversions.java

```diff
@@ -25,7 +25,7 @@ public static double rotationsToOutputDegrees(double rotations, double gearRatio
    *          Gear Ratio between Motor and Mechanism
    * @return Input Shaft Rotations
    */
-  public static double degreesToInputRotations(double degrees, double gearRatio)
+  public static double degreesToOutputRotations(double degrees, double gearRatio)
```

Comment:
Generally, we shouldn't be changing the methods in the Conversions class. This method calculates the number of input rotations on a gearbox required to move a certain number of degrees on its output. If you need a different method, then write one to do the job. But the name of this one should be correct. (degreesToInputRotations)